File: app/src/main/java/com/gmail/uwriegel/tasks/json/GoogleAccount.java
package com.gmail.uwriegel.tasks.json;

/**
 * Created by urieg on 01.05.2017.
 */

public class GoogleAccount {
    public GoogleAccount(String name, String displayName, String photoUrl) {
        this.name = name;
        this.displayName = displayName;
        this.photoUrl = photoUrl;
    }

    public String name;
    public String displayName;
    public String photoUrl;
}

File: app/src/main/java/com/gmail/uwriegel/tasks/json/GoogleAccount.kt
package com.gmail.uwriegel.tasks.json

import android.net.Uri

/**
 * Created by urieg on 01.05.2017.
 */

class GoogleAccount(var name: String, var displayName: String, photoUrl: Uri?) {
    var photoUrl: String = ""

    init {
        if (photoUrl != null)
            this.photoUrl = photoUrl.toString()
    }
}
